### DISCUSSION TITLE
```-m``` command not working

### DISCUSSION THREAD

**Discussion by Lucian-Ryu** [Cyfrin/foundry-full-course-cu#570] [Q&A]
hey guys, quick question , when i try running ```-m``` to run a single test on my console it tells me ```error: unexpected argument '-m' found```
do i need o install anything in particular so that this shortcut works?

thank you

**Comment by shikhar229169** [comment]
It is actually ```--mt```

So you need to run like this:
```bash
forge test --mt test_function_name
```

You can also view it using ```forge test --help```
Over there you will find ```--match-test``` and ```--mt``` is the alias for it.

**Comment by 0xKumarShivam** [comment]
use
```
 --mt 
```
in place of -m
and 
```
--mt -vvvv
```
 for detailed test analysis

**Comment by albin-george-kurian** [comment]
use -mt instead of -m

correct code:
```
forge test -mt FUNCTION_NAME
```
